### ISSUE TITLE
cannot import name 'QtWebChannel' from 'PySide2'

### ISSUE THREAD

**Issue by j-wiedemann** [cadenasgmbh/3dfindit-freecad-integration#5]
Both tool does not work on FreeCAD Daily from PPA-daily on Ubuntu 20.04:
 

```
11:25:55  Running the Python command 'CADENAS3Df_Show' failed:
Traceback (most recent call last):
  File "/home/jo/.FreeCAD/Mod/3DfindIT/freecad/cadenas3dfindit/init_gui.py", line 40, in Activated
    from freecad.cadenas3dfindit import dialog
  File "/home/jo/.FreeCAD/Mod/3DfindIT/freecad/cadenas3dfindit/dialog.py", line 22, in <module>
    from PySide2 import QtWebChannel
```

```
cannot import name 'QtWebChannel' from 'PySide2' (/usr/lib/python3/dist-packages/PySide2/__init__.py)
Running the Python command 'CADENAS3Df_GeoSearch' failed:
Traceback (most recent call last):
  File "/home/jo/.FreeCAD/Mod/3DfindIT/freecad/cadenas3dfindit/init_gui.py", line 57, in Activated
    from freecad.cadenas3dfindit import dialog
  File "/home/jo/.FreeCAD/Mod/3DfindIT/freecad/cadenas3dfindit/dialog.py", line 22, in <module>
    from PySide2 import QtWebChannel

cannot import name 'QtWebChannel' from 'PySide2' (/usr/lib/python3/dist-packages/PySide2/__init__.py)
```

```
OS: Ubuntu 20.04.1 LTS (XFCE/xubuntu)
Word size of OS: 64-bit
Word size of FreeCAD: 64-bit
Version: 0.19.
Build type: Release <--**PPA daily : latest upgrade was 25/11/2020** 
Branch: unknown
Hash: 711b4440f25e2fb24d20d72fe2b8af415d5959b1
Python version: 3.8.5
Qt version: 5.12.8
Coin version: 4.0.0
OCC version: 7.3.0
Locale: French/France (fr_FR)
```

I think QtWebEngine is not present on my system, I have not dig further.
It work ok with daily AppImage version.

**Comment by tsielaff** [comment]
I can reproduce this on my Ubuntu, too. Seems the daily build from freecad-maintainers/freecad-daily doesn't include the QtWebEngine. The plugin, however, requires it. Likely nothing I can do besides providing a better error message. Sorry.

**Comment by berndhahnebach** [comment]
Hi Jonathan, how are you?

See the German topic about the plug in... You are missing some packages. See especially this post https://forum.freecadweb.org/viewtopic.php?f=13&t=52617&start=10#p452302

**Comment by berndhahnebach** [comment]
@tsielaff a better error message in this regard would be good. We already had a few users which could not solve this.

**Comment by berndhahnebach** [comment]
BTW: The daily AppImage version includes the missing packages.
Since this is an AddOn the packages are not a dependency of FreeCAD and get not installed by the package manager if FreeCAD is installed via ppa.

**Comment by tsielaff** [comment]
I agree. I'll implement something so we get a better error message that tell the user to install the missing packages.

**Comment by berndhahnebach** [comment]
What exactly did you use as commit message to get the link to the issue.

**Comment by tsielaff** [comment]
"fix #5"
